frequently-asked-questions.md

```diff
@@ -0,0 +1,90 @@
+
+**Q: I picked out a project. What do I do next?**
+A: Each project comes with a README which guides you on how to setup the project. If you face any problems you need to head over to the project’s Zulip channel and ask your question there. Someone from the community will help you out. If you need help right away, please mention @project-managers and someone will help you out. Once the project is setup, read the contributing guidelines to make your first contribution.
+
+## Open Source Programs
+
+**Q: Can previous ambassadors, apply to be an ambassador again?**
+A: Yes! You can re-apply.
+
+**Q: Do GSoC mentors receive compensation?**
+A: Please email gsoc-support@google.com for inquiries and learn more information on the [Google Summer of Code official website](https://summerofcode.withgoogle.com/).
+
+**Q: Can I contribute to GSoC projects even if I’m not a GSoC student?**
+A: Yes, GSoC projects are discussed in public channels because we want community involvement. You can even attend weekly meetings of each project if you want to! These projects are just as open source as they were before the GSoC selections. You can contribute with ideas, and even issues if they're not labeled as GSoC issues.
+
+**Q: Do I have to be a part of GSoC/Ambassadors/Maintainers to contribute to AnitaB.org Open Source?**
+A: No, you don’t need to be a part of those programs to contribute to AnitaB.org Open Source. Anyone can contribute to the open source community anytime.
+
+**Q: Do I have to be a part of Open Source Ambassadors to contribute to AnitaB.org Open Source?**
+A: No, you do not have to be a part of open source teams to contribute to AnitaB.org Open Source.
+
+## Systers to AnitaB.org Open Source change
+
+**Q: What is the difference between Systers Open Source and AnitaB.org Open Source?**
+A: We rebranded from Systers Open Source to AnitaB.org Open Source in 2020. GitHub projects that were active or still needed development have been moved to [github.com/anitab-org](http://github.com/anitab-org). 
+
+**Q: Are we still using Systers-open source on Slack?**
+A: Nope, we moved to Zulip - [AnitaB-org.zulipchat.com](https://anitab-org.zulipchat.com/) and anyone who was in our Slack was invited to the new workspace.
+
+**Q: Why did we move from Slack to Zulip?**
+A: Here are some reasons:
+- Private Messages were disabled.
+- Age minimum on Slack is 16 years old, which excludes our Google Code-in Participants.
+
```

Comment:
Nothing big. 'Age minimum on Slack is 16 years old' can be written as 'Minimum age required to be on Slack is 16 years'.

Reply:
done! thank you for your review @aditmehta9 🙌